polaris-react/src/components/LegacyCard/LegacyCard.scss

```diff
@@ -219,13 +231,13 @@
 .Footer {
   display: flex;
   justify-content: flex-end;
-  padding: 0 var(--p-space-4) var(--p-space-4);
+  padding: var(--p-space-2) var(--p-space-4) var(--p-space-4);
 
   @media #{$p-breakpoints-sm-up} {
     padding: 0 var(--p-space-5) var(--p-space-5);
 
     #{$se23} & {
-      padding: 0 var(--p-space-4) var(--p-space-4);
+      padding: var(--p-space-2) var(--p-space-4) var(--p-space-4);
```

Comment:
Like .Header, want to treat .Footers the same as a section with `8px` top and bottom, `16px` sides. This makes it simpler to override in javascript